packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -143,12 +138,29 @@ const logRequest = (request: any, {logText = ''}) => {
     });
 };
 
+export type LocalTracks = {
+  microphone?: LocalMicrophoneTrack;
+  camera?: LocalCameraTrack;
+  screenShare?: {
+    audio?: LocalTrack; // todo: for now screen share audio is not supported (will be done in SPARK-399690)
+    video?: LocalDisplayTrack;
+  };
+  annotationInfo?: AnnotationInfo;
```

Comment:
When merging latest master into my branch I've found that this annotation info has been recently added to publishTracks() API (in this PR:https://github.com/webex/webex-js-sdk/pull/2894). I think it doesn't belong here, it should probably be a property on the LocalDisplayTrack itself, but for now I've kept it and will discuss this with the Marmot team next week after their public holidays.